scripts/ci-checks-v2.sh

```diff
@@ -34,7 +40,8 @@ if [[ "${GITHUB_EVENT_NAME:-}" == 'pull_request' ]]; then
       if [[ "${package_name}" == "storage_impl" ]]; then
         all_commands+=("cargo hack clippy --features 'v2,payment_v2,customer_v2' -p storage_impl")
       else
-        all_commands+=("cargo hack clippy --feature-powerset --depth 2 --ignore-unknown-features --at-least-one-of 'v2 ' --include-features '${v2_feature_set}' --package '${package_name}'")
+        valid_features=$(features_to_run "$package_name")
```

Comment:
Nit: Quote usages of variables.

```suggestion
        valid_features="$(features_to_run "${package_name}")"
```